From 636b64caee57e3afbe0c92c723ff266c9d4391b1 Mon Sep 17 00:00:00 2001
From: Matthias Reichl <hias@horus.com>
Date: Sun, 18 Nov 2018 13:21:26 +0100
Subject: [PATCH 210/703] Revert "ASoC: wm8804: MCLK configuration options,
 32-bit"

This reverts commit 3b12dcf797f5a4635aecd7f5c090dc507b124ffd.

Despite the commit message being wrong idle_bias changes
were already reverted in the 4.14 tree.

So drop the commit to bring the wm8804 driver back in line with
the rpi-4.14.y and upstream linux trees.

Signed-off-by: Matthias Reichl <hias@horus.com>
---
 sound/soc/codecs/wm8804.c | 1 -
 1 file changed, 1 deletion(-)

--- a/sound/soc/codecs/wm8804.c
+++ b/sound/soc/codecs/wm8804.c
@@ -550,7 +550,6 @@ static const struct snd_soc_component_dr
 	.use_pmdown_time	= 1,
 	.endianness		= 1,
 	.non_legacy_dai_naming	= 1,
-	.idle_bias_on		= true,
 };
 
 const struct regmap_config wm8804_regmap_config = {
